website/source/docs/commands/connect/envoy.html.md.erb

```diff
@@ -0,0 +1,178 @@
+---
+layout: "docs"
+page_title: "Commands: Connect Proxy"
+sidebar_current: "docs-commands-connect-envoy"
+description: >
+  The connect proxy subcommand is used to run the built-in mTLS proxy for Connect.
+---
+
+# Consul Connect Envoy
+
+Command: `consul connect envoy`
+
+The connect envoy command is used to generate a bootstrap configuration for
+[envoy proxy](https://envoyproxy.io) for use with [Consul
+Connect](/docs/connect/).
+
+The default behaviour is to generate the necessary bootstrap configuration for
+envoy based on the environment variables and options provided and by taking to
+the local Consul agent. It calls `exec` the envoy binary with that configuration
```

Comment:
Grammar here maybe? "`exec` [on] the envoy binary"?.

Also thinking there should the note about PATH here up-front. I think the subcommand should have an obvious sentence above the fold in it that notes that it doesn't embed Envoy but is just exec'ing.

Reply:
I changed this to 

> It `exec`s an external Envoy binary with that
configuration leaving the Envoy process running in the foreground.

I left out `$PATH` just because that's only the default and a custom binary can be used which I don't want to get into in this intro paragraph but added `external` to call that out more clearly.